Sources/Redis/RedisCommandTarget.swift

```diff
@@ -91,6 +91,84 @@ public extension RedisCommandTarget { // Future based
     return _enqueue([ "KEYS", pattern ])
   }
   
+  func scan(cursor: String = "0", pattern: String?, count: Int? = nil)
+    -> EventLoopFuture<(String, [String])>
+  {
+    var req = [
+      "SCAN".toRESPValue(),
+      cursor.toRESPValue()
+    ]
+    
+    if let pattern = pattern {
```

Comment:
It's a little stupid to create an array just to throw it away ;-) Also _if_ you do an array, you can just + it. So either:
```
req += [ "MATCH".toRe..., pattern.toResp... ]
```
or just
```
req.append("MATCH".to....)
req.append(...")
```
to avoid the array allocations.